infrastructure/airy.tpl.yaml

```diff
@@ -3,54 +3,56 @@ global:
   appImageTag: develop
   containerRegistry: ghcr.io/airyhq
   namespace: default
```

Comment:
but isn't  ngrok setting missing here?

Reply:
We can also put it there, but i think It is not needed as the only place we can overwrite it is during the bootstrap. And there is a default `false` inside the chart.